amundsen_application/static/js/components/common/TagsList/index.tsx

```diff
@@ -68,28 +41,36 @@ export class TagsList extends React.Component<TagsListProps, TagsListState> {
   }
 
   render() {
-    let innerContent;
     if (this.props.isLoading) {
-      innerContent = <LoadingSpinner/>;
-    } else {
-      innerContent = (
-        <div id="browse-body" className="browse-body">
-          {this.generateTagInfo(this.state.curatedTags)}
-          {
-            this.state.curatedTags.length > 0 && this.state.otherTags.length > 0 &&
-              <hr />
-          }
-          {this.generateTagInfo(this.state.otherTags)}
+      return <LoadingSpinner/>;
+    }
+    if (AppConfig.browse.curatedTags.length === 0) {
+      return (
+        <div id="tags-list" className="tags-list">
+          { this.generateTagInfo(this.props.allTags) }
```

Comment:
It seems like this is introducing a new feature of showing all existing tags in the `TagsList` if there are no curated tags. Do you feel strongly that this is the correct things to do?
1. I generally think changes to existing behaviors should be more intentional. 
2. Consider that this also affects the `HomePage`. Current users outside of Lyft  who use tags but don't have any curated tags will now have a big block of tags of some unknown length on their homepage.
3. Considering that we have a configuration called `showAllTags`, at the very least this should check that configuration before we actually go ahead and show all the tags. 

My overall sentiment is that I'd prefer to not change anything about this component in this PR beyond what is needed to update the dependencies, *but* if you really feel strongly that we include a configuration check for whether or not `showAllTags` is desired.

Reply:
I can revert this behavior, but in my personal opinion we should probably show all tags on the browse page no matter what and show curated or some subset of tags on the home page.

Reply:
IMO I prefer to show all tags in the browse page if possible. Currently there is no way to tell what are the tags are available...

And the browse page is a bit duplicate in function with the home page tags.

Reply:
Cool. I'm open to revisiting the behavior as part of a dedicated PR that explores how to make `BrowsePage` more useful, but we can have this current PR go through with only changes needed to make us compatible with the library upgrades. 

Want to discuss offline more? I do feel like the `BrowsePage` should still have some kind of logic that allows the developers to not to show all tags if they don't want to -- their ecosystem could be full of low usage tags that don't provide much benefit. Also have thoughts on still limiting the initial amount of tags shown, but ordering them by count and creating a UI where users can expand to see more and more if they want.

Reply:
we could discuss more tomorrow in our standup, but  I think ordering by count makes sense and we should expose more internal metadata for the tool in the browse page(e.g what are the tables got index, how tables we index, what are the db/system(bq,rs,presto,hive) we index). These infos are all valuablable but currently hidden somewhere in the backend from users.